docs/GettingStarted.md

```diff
@@ -2138,7 +2138,6 @@ config.lograge.custom_options = lambda do |event|
       :version => correlation.version.to_s
     },
     :ddsource => ["ruby"],
```

Comment:
I agree with the feedback, though i do think it's good to show the user that they can reference the `event.payload` within the custom_options block. So suggesting to repace `:params` with some more benign, like `:status`
```suggestion
    :ddsource => ["ruby"],
    :status => event.payload[:status]
```